### DISCUSSION TITLE
CGAL 5.6 beta 1

### DISCUSSION THREAD

**Discussion by stromhaug-tommy** [CGAL/cgal#7530] [Q&A] [answered]
Hi,

When installing version 5.6 beta 1 with the Vcpkg(https://doc.cgal.org/5.6-beta1/Manual/windows.html#title0) a lot of header files where missing.When filling .h files from https://github.com/CGAL/cgal/releases/tag/v5.6-beta1 i somehow got something to work.
maybe the Vcpkg package really is not 5.6?
I have trouble building Remesh_Almost_Planar_Patches.
Anybody who knows how to get rid of these (Not a member of CGAL for all the red ones)?
![image](https://github.com/CGAL/cgal/assets/62478982/120757d3-2364-4fc5-a3f8-1623cb2bbbb3)

**Comment by lrineau** [comment, marked as answer]
Hi,

For the moment the CGAL package in vcpkg is still version 5.5.2. I will submit CGAL version 5.6 to vcpkg only after the beta-testing phase.
